### DISCUSSION TITLE
Components to collect feedback, evaluations and reactions for your pages

### DISCUSSION THREAD

**Discussion by giuseppelt** [facebook/docusaurus#9054] [Show and tell]
Hi, I'm building [Feelback](https://www.feelback.dev/) which provides `Components` to collect feedback on your pages.

I wrote a [guide to use a system like the one in the Stripe documentation](https://www.feelback.dev/docs/guides/stripe-like-documentation-feedback-for-docusaurus) in your Docusaurus website.

![](https://www.feelback.dev/images/posts/stripe-docs-feedback-question.png)
![](https://www.feelback.dev/images/posts/stripe-docs-feedback-choices.png)

Tutorials for other scenarios:

- How to add a 'Send Feedback` button [Tutorial](https://www.feelback.dev/docs/guides/feedback-button-for-docusaurus)
- Upvote buttons for support/requests pages [Tutorial](https://www.feelback.dev/docs/guides/page-useful-upvote-button-for-docusaurus)
- Like buttons for a blog post [Tutorial](https://www.feelback.dev/docs/guides/like-button-for-docusaurus)

Other `Components` you can use

![](https://www.feelback.dev/images/hero-img.png)

Hope you find it useful. I'm available for help or any suggestions.

**Comment by slorber** [comment]
Hi

Yes that's interesting for sure, had the idea for such a product in the past.

Historically some users have used GA for that, see for example https://reactnative.dev/

![CleanShot 2023-06-09 at 16 40 23@2x](https://github.com/facebook/docusaurus/assets/749374/fc3d6a8f-ee58-44ab-835b-aafc8b66f531)


Having a tool dedicated to this problem, in particular for content-centric sites can improve the UX and make it simpler to use than GA


Considering documentation can be versioned and/or localized, that could be interesting to be able to compare ratings of docs regarding these parameters.

**Reply by giuseppelt** [reply]
Hi, thanks!
For doc websites, I think a with _boolean signal_ doesn't do much. I think, GA is used to due to lack of a better alternative.

For example, like in the stripe feedback, with **Feelback** you can define tags and assign sentiment values:

![feedback-tags](https://github.com/facebook/docusaurus/assets/7147291/5039231a-49eb-4e5c-a390-c80d995f9093)

Then you can checkout aggregates over time

![chart-feedback-tags](https://github.com/facebook/docusaurus/assets/7147291/ce27c4e2-1ae8-41c6-a244-cf0ea870eaa5)



Since, I'm early stage right now, if you have any thought or feature you may think can be useful, I'm collecting suggestions to shape the product.
Many thanks.
